machine-learning/pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "machine-learning"
+version = "1.59.1"
```

Comment:
Is this something we'll need to bump each release? If so can you take a look at the pump script?

Reply:
Ooh, I didn't notice that script. It should work for this file too now but only if it's used from the project root.